magenta/models/latent_transfer/interpolate_joint.py

```diff
@@ -23,15 +23,15 @@
 from __future__ import print_function
 
 import importlib
-from os.path import join
-
-import numpy as np
-import tensorflow as tf
+import os
 
 from magenta.models.latent_transfer import common
 from magenta.models.latent_transfer import common_joint
 from magenta.models.latent_transfer import model_joint
+import numpy as np
+import tensorflow as tf
 
+join = os.path.join
```

Comment:
There are few enough calls that I would prefer to just use the full `os.path.join` for each call.